aws-cdk/lib/s3-stack.ts

```diff
@@ -0,0 +1,16 @@
+import * as cdk from 'aws-cdk-lib';
+import * as s3 from 'aws-cdk-lib/aws-s3';
+import { Construct } from "constructs";
+
+export class BucketResourceStack extends cdk.Stack {
+    constructor(scope: Construct, id: string, props?: cdk.StackProps) {
+        super(scope, id, props);
+
+        new s3.Bucket(this, 'waectbucket', {
+            bucketName: 'waectbucket',
+            publicReadAccess: true, // Allow public read access
+            blockPublicAccess: s3.BlockPublicAccess.BLOCK_ACLS, // Allow public access by disabling ACL blocking
```

Comment:
Good question, I looked into it. A bit complicated and weird but basically `publicReadAccess` enables the S3-bucket to have public-read-access and since S3-buckets by default blocks public reads we also need to disable `blockPublicAccess`. 

`publicReadAccess`  is legacy and have been kept for some reason but `blockPublicAccess` is new and needs to unblocked if we want it to be fully public :))